### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by devalih**
### Description

TASK: [ASANA LINK](https://app.asana.com/X/XXXXXX/XXXXXX)

### CODE REVIEW

File: conf/locale/rtl/LC_MESSAGES/djangojs.po [hunk 1]
```diff
@@ -7798,10 +7796,6 @@ msgstr "حهررثي"
 msgid "Following"
 msgstr "بخممخصهرل"
```
Comment (halawa) [inline conf/locale/rtl/LC_MESSAGES/djangojs.po:7797]: Please confirm

File: conf/locale/rtl/LC_MESSAGES/djangojs.po [hunk 2]
```diff
@@ -7730,12 +7731,9 @@ msgid "post anonymously to classmates"
...
 #: common/static/common/templates/discussion/post-user-display.underscore
-msgid "(Community TA)"
-msgstr "(ذخووعرهفغ فش)"
-
-#: common/static/common/templates/discussion/post-user-display.underscore
-msgid "(Staff)"
-msgstr "(سفشبب)"
+#: common/static/common/templates/discussion/thread-list-item.underscore
+msgid "Community TA"
+msgstr "ذخووعرهفغ فش"
```
Comment (halawa) [inline conf/locale/rtl/LC_MESSAGES/djangojs.po:7736]: Please confirm

File: conf/locale/eo/LC_MESSAGES/djangojs.po [hunk 3]
```diff
@@ -8642,12 +8643,9 @@ msgid "post anonymously to classmates"
...
 #: common/static/common/templates/discussion/post-user-display.underscore
-msgid "(Community TA)"
-msgstr "(Çömmünïtý TÀ) Ⱡ'σяєм ιρѕυм ∂σłσя ѕιт#"
-
-#: common/static/common/templates/discussion/post-user-display.underscore
-msgid "(Staff)"
-msgstr "(Stäff) Ⱡ'σяєм ιρѕυм #"
+#: common/static/common/templates/discussion/thread-list-item.underscore
+msgid "Community TA"
+msgstr "Çömmünïtý TÀ Ⱡ'σяєм ιρѕυм ∂σłσя ѕ#"
```
Comment (halawa) [inline conf/locale/eo/LC_MESSAGES/djangojs.po:8648]: Please confirm

File: conf/locale/eo/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -24876,6 +24854,10 @@ msgstr ""
 msgid "Explore New Courses"
 msgstr "Éxplöré Néw Çöürsés Ⱡ'σяєм ιρѕυм ∂σłσя ѕιт αмєт,#"
 
+#: openedx/features/learner_profile/templates/learner_profile/learner_profile.html
+msgid "Learner Profile"
+msgstr "Léärnér Pröfïlé Ⱡ'σяєм ιρѕυм ∂σłσя ѕιт α#"
+
```
Comment (halawa) [inline conf/locale/eo/LC_MESSAGES/django.po:24860]: Please confirm

File: conf/locale/rtl/LC_MESSAGES/djangojs.po (hunk 2 above)
Comment (devalih) [reply conf/locale/rtl/LC_MESSAGES/djangojs.po:7736]: Random generated for testing rtl

File: conf/locale/eo/LC_MESSAGES/djangojs.po (hunk 3 above)
Comment (devalih) [reply conf/locale/eo/LC_MESSAGES/djangojs.po:8648]: we don't use EO languge code

File: conf/locale/eo/LC_MESSAGES/django.po (hunk 4 above)
Comment (devalih) [reply conf/locale/eo/LC_MESSAGES/django.po:24860]: we don't use EO languge code

**Review by halawa — APPROVED** [review]

**Pull request merged**